server/athenian/api/controllers/miners/github/release_load.py

```diff
@@ -665,11 +684,16 @@ def match_groups_to_sql(match_groups: Dict[ReleaseMatch, Dict[str, Iterable[str]
     ]:
         if not (match_group := match_groups.get(match)):
             continue
-        or_items.extend(and_(model.release_match == "".join([match.name, suffix, v]),
-                             model.repository_full_name.in_(r))
-                        for v, r in match_group.items())
+
+        or_conditions.extend([
+            {
+                "release_match": "".join([match.name, suffix, v]),
+                "repository_full_name": r,
+            } for v, r in match_group.items()
+        ])
```

Comment:
```suggestion
        or_conditions.extend({
            "release_match": "".join([match.name, suffix, v]),
            "repository_full_name": r,
        } for v, r in match_group.items())
```
(more compact formatting + no need to allocate a list to extend)